domain/optatask/pom.xml

```diff
@@ -0,0 +1,56 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.optatask</groupId>
+  <artifactId>optatask</artifactId>
+  <packaging>kjar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>optatask</name>
+
+  <properties>
+    <optaplanner.version>7.12.0.Final</optaplanner.version>
+    <kie.version>7.12.0.Final</kie.version>
```

Comment:
It's better to use "version" in the beginning of the property name so when there are more properties (not just version properties), the version properties are grouped together. It's also a good practice to keep properties sorted alphabetically. See https://github.com/kiegroup/optaweb-employee-rostering/blob/master/pom.xml#L44-L83.